backend/typescript/rest/campRoutes.ts

```diff
@@ -27,7 +29,7 @@ const campService: ICampService = new CampService(fileStorageService);
 campRouter.get("/", async (req, res) => {
   try {
     const camps = await campService.getCamps();
-    res.status(200).json(camps);
+    res.status(200).send(camps);
```

Comment:
why did we change this?

Reply:
not gonna lie, no idea. I will just change it back lol, I don't think it makes a diff though.